daemon/dns/dns_resolvectl.go

```diff
@@ -27,7 +27,16 @@ func (m *Resolvectl) Unset(iface string) error {
 }
 
 func (m *Resolvectl) IsAvailable() bool {
-	return internal.IsCommandAvailable(execResolvectl)
+	if internal.IsCommandAvailable(execResolvectl) {
```

Comment:
[minor] wouldn't be enough to have only the second if, without checking if the command exists before? because in the end the command is always executed, and if it doesn't exist will just return error.

Reply:
yes, good point, thx ;)